system/Database/MySQLi/Result.php

```diff
@@ -58,47 +87,51 @@ public function getFieldNames(): array
 	 */
 	public function getFieldData(): array
 	{
-		static $dataTypes = [
+		static $data_types = [
```

Comment:
Why snake_case?

Reply:
I had mistakenly copied MySQLi/Result.php from a project I've been working on for a few weeks and I guess there were extensive edits to the comments (and this change of var name) in the interim. It became apparent after I submitted this PR that I needed to use more up-to-date files. That is why I closed this PR.